src/providers/google-vertex-ai/chatComplete.ts

```diff
@@ -269,6 +269,10 @@ export const VertexGoogleChatCompleteConfig: ProviderConfig = {
       const tools: any = [];
       params.tools?.forEach((tool) => {
         if (tool.type === 'function') {
+          // these are not supported by google
+          delete tool.function?.parameters?.properties?.additional_properties;
```

Comment:
I think its also possible to have additional_properties in nested object. So we would have to handle that as well. Can you please verify once?

Reply:
done!